server/index.spec.js

```diff
@@ -0,0 +1,65 @@
+const supertest = require('supertest');
+const { app } = require('./index.js');
+const request = supertest(app);
+
+describe('/:listingId', () => {
+
+  it('should serve static files at listingIds 0-99', async (done) => {
+
+    const res = await request.get('/32')
+
+    expect(res.status).toBe(200)
+    expect(res.text).toContain('html')
+    done()
+  })
+
+  it('should return a 404 error for any pages not at listingId 0-99', async (done) => {
```

Comment:
Good idea to test listingIds outside of our predetermined range. I should do that....